billboard.py

```diff
@@ -65,8 +66,17 @@ class ChartEntry:
         rank: The track's position on the chart, as an int.
         isNew: Whether the track is new to the chart, as a boolean.
```

Comment:
Can you add the new field to the doc comment?

Reply:
yes, will do